flask_multipass/core.py

```diff
@@ -135,7 +135,8 @@ def validate_next_url(self, url):
         a whitelist of trusted hosts to avoid creating an open redirector.
         """
         url_info = urlsplit(url)
-        if url_info.scheme and url_info.scheme not in {'http', 'https'}:
+        if (url_info.scheme and url_info.scheme not in {'http', 'https'})\
```

Comment:
This will be much more readable if you put it into a second `if` statement.

Reply:
Indeed, I'll update the PR.